### PR TITLE
Add Top section solves #56

### LINKED ISSUE #56: Top section
### How it should look like?
Top FOSSASIA section with partners like on the most FOSSASIA websites, e.g. https://2012.fossasia.org/ with the content of sidebar

### How it looks now?
Now we just have a sidebar with no top section

### Would you like to work on it?
Yes! 😄

### PR DESCRIPTION
**Pull request by kubami9**
### Add top section similar to most FOSSASIA websites. Fix #56 

**This PR includes:**
 - full width header with FOSSASIA background-image (the same which is on FOSSASIA Summit 2012 and 2018)
 - partners/sponsors section which includes all content which has been in the sidebar, apart from the Flickr images
 - delete sidebar
 - FOSSASIA newsletter section (basically email subscription from sidebar turned into own section, somewhat similar to this from fossasia.org)
 - center content
 - menu background changed to transparent and font color to white
 - some simple bug and typo fixes

Preview link: https://kubami9.github.io/2014.fossasia.org-km-contribute/

### CODE REVIEW

**Review by isuruAb — CHANGES_REQUESTED** [review]
Hello @kubami9  there are some changes to be done.
logo is missing and font size should be little bit smaller. please follow the styles of fossasia.org and don't forget to squash your commits.
cheers

**Comment by kubami9** [comment]
@isuruAb done 😄

File: _includes/header.html [hunk 1]
```diff
@@ -1,52 +1,61 @@
...
 			font-size: 1.4em;
 			text-transform: none;
 			letter-spacing: 0;
 			padding: 0px;
+		}
 
+		h1 {
+			font-family: 'Open Sans';
+			font-weight: normal;
+      		text-transform: uppercase;
```
Comment (codedsun) [inline _includes/header.html]: The indentation needs to be corrected

**Comment by kubami9** [comment]
@isuruAb could you check out my changes 😄 this PR is getting more and more conflicts to resolve and whenever I resolve them I need to squash and it all loops

**Review by ParthS007 — APPROVED** [review]

**Review by akshatnitd — COMMENTED** [review]
The changes looks good. You just need to fix the indentation issues at various lines, keeping in view the best practices. And, lastly squash the commits! The, it would be good to go :+1:

**Comment by kubami9** [comment]
@akshatnitd Is it good now?

**Review by nzec — APPROVED** [review]
There are a lot of problems but I think they can be improved over time with different issues. 

I think this should be merged first.

@kubami9 Awesome work! 👍

File: _includes/header.html [hunk 2]
```diff
@@ -389,20 +369,117 @@
 a:hover {
 	text-decoration: underline;
 }
-#feet{
-	display:flex;
-	wrap: nowrap;
-
 
+  .top {
+  background-image: url(images/fossasia2016.jpg);
```
Comment (akshatnitd) [inline _includes/header.html]: Missing indentation here and same in many lines below. Also, I can see 2 commits, make sure only 1 commit remains after squashing.

**Review by akshatnitd — CHANGES_REQUESTED** [review]

File: _includes/header.html [hunk 3]
```diff
@@ -208,14 +180,21 @@
 	font-family: "open sans";
 }
 
+#menu {
+  	width: 100%;
+}
 /*Strip the ul of padding and list styling*/
 ul {
+<<<<<<< HEAD
```
Comment (nzec) [inline _includes/header.html]: Fix merge errors.

**Comment by kubami9** [comment]
It's confusing, whenever I squash commits it gets conflicts and whenever I resolve conflicts it needs to be squashed

**Comment by kubami9** [comment]
And now - I don't even know why - it shows me that I changed 93 files

**Comment by kubami9** [comment]
@akshatnitd  Now it should be okay, however I can't merge it into just one commit since it's giving dozens of conflicts

**Review by akshatnitd — APPROVED** [review]
Good work! :) Merging :rocket: 
Further improvements can be done by next PRs.

**Comment by kubami9** [comment]
Yaay! Thank you so much @akshatnitd!!! 😃

**Pull request merged**